### PR TITLE
Merge milestone-2 branch into development branch 

### PR DESCRIPTION
**Pull request by Laguna1**
What this PR does:

- Prepared the directory structure
-- Created a `components`  directory in the `src` folder
-- Created four files into the `components` folder - `App.js`, `Display.js`, `ButtonPanel.js` and `Button.js` - according to requirements
- Implemented the App component
-- App renders the Display and ButtonPanel component. 
- Implemented the Display component
-- It accepts the result of the calculation as an incoming prop.
-- The resulting prop is a String
-- The default value of the resulting prop equals 0.
- Implemented the Button component
-- It accepts a button name as a prop.
-- The button name prop is a String.
- Implemented the ButtonPanel component
-- It renders the calculator panel with all the buttons in the five groups (used divs to divide buttons into groups)
- The results of the work are page full of non-functional buttons and it was updated on Heroku at https://calculator-oks.herokuapp.com/

### CODE REVIEW

File: src/components/App.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import '../App.css';
+import Display from './Display';
+import ButtonPanel from './ButtonPanel';
+
+function App() {
+  return (
+    <div className="App">
+      <Display result="something display" />
+      <ButtonPanel />
+    </div>
```
Comment (MuriloRoque) [inline src/components/App.js]: - [ ] **(OPTIONAL)** You can use [React Fragment syntax](https://reactjs.org/docs/fragments.html#short-syntax) here if you don't want this `div`.

File: src/components/Button.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+import PropTypes from 'prop-types';
+
+const Button = props => {
+  const { buttonName } = props;
```
Comment (MuriloRoque) [inline src/components/Button.js:4]: - [ ] **(OPTIONAL)** You can make this code even shorter by simply destructuring the props directly in the function's argument :smile:

File: src/components/Display.js [hunk 3]
```diff
@@ -0,0 +1,16 @@
+import PropTypes from 'prop-types';
+
+const Display = props => {
+  const { result } = props;
```
Comment (MuriloRoque) [inline src/components/Display.js:4]: - [ ] **(OPTIONAL)** You can make this code even shorter by simply destructuring the props directly in the function's argument :smile:

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @Laguna1 :wave: 

### Amazing work in this milestone, your code is simple and efficient, but I still gave you some tips to make it even better (check my comments in your code). Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

File: src/components/App.js (hunk 1 above)
Comment (Laguna1) [reply src/components/App.js]: @muriloroque thank you a lot for the review

**Pull request merged**